libraries/AP_GPS/AP_GPS_PX4.h

```diff
@@ -36,8 +36,13 @@ class AP_GPS_PX4 : public AP_GPS_Backend {
 
     const char *name() const { return "PX4"; }
 
+protected:
+    void broadcast_gps_type() const override;
+
 private:
     int                           _gps_sub;
     struct vehicle_gps_position_s _gps_pos;
+
+    bool _message_seen = false;
```

Comment:
_Size reduction gang_: the initialization shouldn't needed, it should be set to false on construction.